### DISCUSSION TITLE
<Error Fetching Changes>

### DISCUSSION THREAD

**Discussion by ARDIGplanning** [community/community#54661] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am trying to publish a commit from GitHub Desktop, and I have to fetch any changes first for it to let me push. When I try to fetch I get this error:

From https://github.com/...
 - [deleted]           (none)     -> origin/HEAD 2
remote: Enumerating objects: 17, done.        
remote: Counting objects: 100% (17/17), done.        
remote: Compressing objects: 100% (12/12), done.        
remote: Total 12 (delta 8), reused 0 (delta 0), pack-reused 0        
fatal: bad object refs/remotes/origin/HEAD 2
error: https://github.com/... did not send all necessary objects

**Comment by Reemaalmunif** [comment]
hi, did you find a solution?
